main/job-counter.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const Store = require('electron-store')
+
+const jobCounterStore = new Store({
+  name: 'job-counters'
+})
+
+class JobCounter {
+  #moduleCounters
+
+  constructor () {
+    this.#moduleCounters = loadStoredCounters()
+  }
+
+  getTotalJobCount () {
+    return Object.values(this.#moduleCounters).reduce((sum, value) => sum + value, 0)
+  }
+
+  /**
+   * @param {string} moduleName
+   * @param {number} count
+   */
+  setModuleJobCount (moduleName, count) {
+    this.#moduleCounters[moduleName] = count
+    this.#save()
+  }
+
+  reset () {
+    this.#moduleCounters = {}
+    this.#save()
+  }
+
+  #save () {
+    storeCounters(this.#moduleCounters)
+  }
+}
+
+/**
+ * @returns {import('./typings').ModuleJobCount}
+ */
+function loadStoredCounters () {
+  // A workaround to fix false TypeScript errors
+  return /** @type {any} */(jobCounterStore.get('counters', {}))
+}
+
+/**
+ * @param {import('./typings').ModuleJobCount} counters
+ */
+function storeCounters (counters) {
+  jobCounterStore.set('counters', counters)
+}
+
+module.exports = {
+  JobCounter
```

Comment:
why not make this the default export?

Reply:
I prefer to use named exports as they are more future-proof. Adding the second named export requires no changes to existing code. Switching from a single default export to two named export requires changes in all places that were using the default export.

Reply:
ok!